resume.md

```diff
@@ -1,49 +1,58 @@
-# FIRST LAST NAME
+# Tejesh Bhaumik
 
-(123) 456 - 7890 | email@gmail.com
+(123) 456 - 7890 | tejeshbhaumik@gmail.com
 
-www.website.com | github.com/myusername | linkedin.com/myusername
+| github.com/Tejesh2001 | linkedin.com/TejeshBhaumik
 
 ## EDUCATION
-**University of College** | B.S. Degree ... *MONTH 20XX*
-* **Coursework**: Data Structures, Algorithms, Upper Division CS Elective, Math Course, Statistics, Data Science, ETC.
+**University of College** | B.S. of Computer Science + Advertising... *May 2023*
+* **Coursework**: Data Structures, Algorithms, Advanced Linear Algebra, Statistics, Data Science, Probability, Brand Strategy.
 
 ## SKILLS
-* **Languages**: Python, Java, C, SQL, ...
-* **Libraries/Frameworks**: Pandas, TensorFlow, PyTorch, ...
-* **Version Control**: Git, Bitbucket, Sourcetree, GithubDesktop
+* **Languages**: Python, Java, C++, SQL, NodeJs, Android
+* **Libraries/Frameworks**: Pandas, Numpy, Scipy, Sympy
+* **Version Control**: Git, GithubDesktop
 * *...*
 
 ## EXPERIENCE
-**Software Engineer**: Company | City, State ... *Month 20XX - Present*
-* Implemented XYZ Microservice using TECHNOLOGY to delivery PRODUCT to 123 Customers Globally
-* Architected CORE INFRASTRUCTURE LIBRARY used across X teams to BLAH BLAH
-* I did X for Y to accomplish Z
-* ...
+**Android Developer**: Crowd Dynamics Lab   | Remote *September 2020 - Present*
+* Assist in the implementation of an Android tracer app using IoT principles
+
+
+**Backend Developer**: Ensemble Education | Remote *August 2020 - October 2020*
+* Implemented social media logins, file systems and assisted in building REST APIS in NodeJS and C++ 
+* Assisted in the development of the AI module using Python using scikit-learn, Pandas and Numpy.
+
+**Software Research Assistant**: UIUC advertising department   | Remote *July 2020 - August 2020*
+* Developed an AI chatbot in Node.js with a user interface in HTML/CSS. Has a user base of 200 surveyors.
```

Comment:
Good use of metrics here.  What is the purpose of this project?  And why does it matter?  Focus on the impact